katunold/express-phone-number-generator
# #167556005: implement generate contacts program logic

### What does this PR do?
- This PR implements the program logic to handle generating a range of unique phone contacts.
- Write unit tests integration tests for the current functionality
- Add compatibility for es6 imports to the codebase
### Description of the task to be completed
_Have the following endpoint working_
1. `GET /generate` to generate a list of unique phone numbers and store them in a JSON file
### How should this be manually tested?
After cloning the repo, `cd` into `express-phone-number-generator` and follow these steps
- Open the codebase using your preferred editor
- In a terminal, type `git checkout ft-generate-contacts-167556005` to switch branch
- The install all dependencies by typing `npm i` in a terminal
- Then type `npm start` to run the server on port `4000`
- Go to either postman or your browser in the url section and paste this url `http://localhost:4000/generate`
- In your browser, you will see a welcome message from express and then check in the current directory to see a file named `contacts.json` created with over `10000` contacts generated.
### Relevant pivotal tracker stories
[167556005](https://www.pivotaltracker.com/story/show/167556005)
